grafana/grafana
# Loki: Added support for disabled operations in Query Builder

This PR adds support to disable operations in the query builder. Operations maked as "toggleable" will disaplay an "eye" icon to toggle the operation active/inactive.

https://github.com/user-attachments/assets/8d31d221-89d9-40ad-b36f-fa416597a89f

Requires https://github.com/grafana/grafana-experimental/pull/135
Closes https://github.com/grafana/grafana/issues/63785

## How to test

In grafana-experimental:
- Checkout https://github.com/grafana/grafana-experimental/pull/135 
- In grafana-experimental, change the following dependency versions:
  ```
    "@hello-pangea/dnd": "17.0.0",
    "@types/uuid": "9.0.8",
    "uuid": "9.0.1"
  ```
    This change has not been commited to not silently break other grafana-experimental features.
- `yarn dev`

In grafana
- Run `yarn link ../path-to-grafana-experimental`
- `yarn start`

You will see a lot of TypeScript errors in the console, which you can ignore.
